lms/djangoapps/grades/migrations/0017_auto_20190924_1411.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: it'd be nice to give this file a more descriptive name (which you can do with the `-n` flag of the django `makemigrations` command).